README - lab30.md

```diff
@@ -0,0 +1,55 @@
+# Labs DS template
+# Asylum Decision  -- Human Rights First
+An application to assist immigration advocates in winning asylum cases
+
+
+## Description
+We built an application for human rights first, a 501(c)3 organization. Our application uses optical character recognition to scan input court decisions for such values as the name of the presiding judge, the decision, and the asylum seeker's country of origin, and inserts these values into a database. The hope is that advocates for asylum seekers can use these data to better tailor their arguments before a particular judge and maximize their client's chances of receiving asylum. Based on our stakeholder meeting HRF wants to know the trend and exploratory of the judge's decision, for example:
+
+ *How many procent is the case rejected with certain nationality or gender?
+ *Is there any trend based on religion or social group?
+ *Is certain judge in favor of an applicant from a certain case? or country?
+
+In order to answer those questions, we need to extract those keywords and put them into dataframe, and from there we store them
+in our database.
```

Comment:
Excellent addition to project README.md because it covers whole group work and speaks to general process

Thank you!